.buildkite/scripts/run_models.sh

```diff
@@ -17,4 +17,4 @@ cd integration_tests
 dbt deps
 dbt seed --target "$db" --full-refresh
 dbt run --target "$db" --full-refresh
-dbt test --target "$db"
\ No newline at end of file
+dbt test --target "$db"dbt run-operation fivetran_utils.drop_schemas_automation --target "$db"
```

Comment:
```suggestion
dbt test --target "$db"

dbt run-operation fivetran_utils.drop_schemas_automation --target "$db"
```

Reply:
Is there a way we can add an additional line here?

Reply:
ah yeah lemme adjust the function to insert a `\n` beforehand